src/request/caches/mod.rs

```diff
@@ -1,17 +1,26 @@
 pub mod modes;
 
-use crate::request::caches::modes::LocalCache;
+use crate::request::caches::modes::*;
 use crate::request::{Method, Request};
 use serde::{Deserialize, Serialize};
 use serde_json::json;
 use std::collections::HashMap;
 
 pub(crate) const CACHES_ENDPOINT: &str = "/rest/v2/caches";
 
-#[derive(Serialize, Deserialize)]
-enum Cache {
+#[derive(Eq, PartialEq, Debug, Serialize, Deserialize)]
+pub enum Cache {
     #[serde(rename = "local-cache")]
-    LocalCache(LocalCache),
+    Local(Local),
```

Comment:
The change disables the ability to use `#[serde(rename_all = "kebab-case")]` on the struct - but it is ok to avoid the `Cache` suffix!

Reply:
Yes, in this case I'd rather avoid the duplicated suffixes :+1: